src/components/button/index.md.njk

```diff
@@ -64,7 +64,7 @@ For example, the GOV.UK Notify team discovered that a number of users were recei
 
 If you are working in production and research shows that users are frequently sending information twice, you can configure the button to ignore the second click. 
 
-To do this, include the `preventDoubleClick` attribute as shown in this example.
+To do this, set the `data-prevent-double-click` attibute to `true`. You can do this directly in the HTML or, if you’re using Nunjucks, you can use the Nunjucks macro as shown in this example.
```

Comment:
typo: attibute

Reply:
I think I already fixed this...

Reply:
I think Joe is reviewing an outdated version of your changes and needs to refresh his browser.

Reply:
sorry, cache issue